src/Desktop/ColorContrastAnalyzer/AnalyzerVersion.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Axe.Windows.Desktop.ColorContrastAnalyzer
+{
+    public enum AnalyzerVersion
+    {
+        Default, // Use whatever the current default happens to be
+        V1,      // Decides based on a single scanline
```

Comment:
It looks like `GetBinaryRowSearchIterator()` inside `ImageCollection.cs` looks at at least one scanline - its comments say: "A special iterator, that looks at the middle of an image first, followed by recursively looking at the upper half and lower half of the two pieces of image, until the given samples are some distance apart." If I'm reading that right, we may want to amend this description.

Reply:
The existing code may scan multiple lines before making its decision, but the ultimate decision is based on exactly one scanline. I updated the comment a bit. If the V2 scanning works out, I'd eventually like to remove the V1 analyzer completely. If perf proves to be a problem, I could also see a world where we dynamically choose one analyzer or the other, possibly based on the size of the bitmap in question